distro/configuration/ampathforms/hiv_and_art_care.json

```diff
@@ -1261,6 +1271,10 @@
                   {
                     "label": "5i = ABC/3TC +LPV/r",
                     "concept": "50b60d77-186d-4a0d-8784-659ee2d60ec9"
+                  },
+                  {
+                    "label": "5j: AZT/3TC (120/60) + DTG10",
```

Comment:
Make this label format consistent with the rest

Reply:
@moshonk this is resolved